quinn/Cargo.toml

```diff
@@ -31,7 +31,7 @@ azure-devops = { project = "dochtman/Projects", pipeline = "Quinn", build = "1"
 bytes = "0.5.2"
 err-derive = "0.2.3"
 futures = "0.3.1"
-libc = "0.2.49"
+libc = "0.2.67"
```

Comment:
But then you don't actually use the new constants from libc?

Reply:
We use `libc::IPV6_MTU_DISCOVER`, which isn't in 0.2.49. The others will come later. Squashed into the relevant commit for clarity.